_html/scss/modules/_homepage-burger-menu.scss

```diff
@@ -71,20 +71,51 @@
     font-size: 1.3rem;
     text-align: center;
 
-    li {
-        padding: .5rem 0;
-
-        span {
+    .menu-item {
+        padding: .7rem 0;
+
+        .action-link {
+            position: relative;
+            color: $white;
+
+            &::after {
+                @include vertical-center;
+                position: absolute;
+                right: -2rem;
+                width: .875rem;
+                height: .5rem;
+                background: url('../images/icons/arrow-up.png') no-repeat;
+                background-position: center;
+                background-size: cover;
+                content: '';
+            }
+            
             @include on-events {
                 color: $secondary-color;
             }
         }
 
+        .active-burger-link {
+            color: $secondary-color;
+            text-decoration: underline;
+
+            &::after {
+                @include vertical-center;
+                position: absolute;
+                right: -2rem;
+                width: .875rem;
+                height: .5rem;
+                background: url('../images/icons/menu-dropdown.png') no-repeat;
+                background-position: center;
+                background-size: cover;
```

Comment:
Please combine background position and background size with general background added on line 108.